examples/mocking/mocking.js

```diff
@@ -1,27 +1,19 @@
-/*
-Example express app that proxies all incoming requests to a backend service and renders the result
- */
+const path = require('path');
 const express = require('express');
-const handlebars = require('express-handlebars');
 
-const {backendProxy, mockBackendResponse, renderBackendResponse} = require('@springernature/backend-proxy');
+const {mockBackendResponse} = require('@springernature/backend-proxy');
 
-const app = new express();
+const app = express();
 
-app.engine('handlebars', handlebars());
-app.set('view engine', 'handlebars');
+app.use(mockBackendResponse({
+	directory: path.resolve('./mocks')
+}));
 
-if (process.env.NODE_ENV === 'development') {
-	console.log('Adding mocking');
-	app.use(mockBackendResponse({
-		directory: "./mocks"
-	}));
-}
-
-app.use(backendProxy({backend: 'http://localhost:8080/api'}));
-app.use(renderBackendResponse());
+app.use((request, response) => {
```

Comment:
Here we remove the `if (process.env.NODE_ENV === 'development') {` check. What was the reasoning behind it?

Reply:
I was thinking that as it's an example we should show it being used all the time. But I think you're correct, that we should highlight that it's to be used in development only